static/js/lib/jquery-migrate-1.2.1.js

```diff
@@ -251,28 +251,28 @@ jQuery.uaMatch = function( ua ) {
 	};
 };
 
-// Don't clobber any existing jQuery.browser in case it's different
-if ( !jQuery.browser ) {
-	matched = jQuery.uaMatch( navigator.userAgent );
-	browser = {};
-
-	if ( matched.browser ) {
-		browser[ matched.browser ] = true;
-		browser.version = matched.version;
-	}
-
-	// Chrome is Webkit, but Webkit is also Safari.
-	if ( browser.chrome ) {
-		browser.webkit = true;
-	} else if ( browser.webkit ) {
-		browser.safari = true;
-	}
-
-	jQuery.browser = browser;
-}
+//// Don't clobber any existing jQuery.browser in case it's different
+//if ( !jQuery.browser ) {
+//	matched = jQuery.uaMatch( navigator.userAgent );
+//	browser = {};
+//
+//	if ( matched.browser ) {
+//		browser[ matched.browser ] = true;
+//		browser.version = matched.version;
+//	}
+//
+//	// Chrome is Webkit, but Webkit is also Safari.
+//	if ( browser.chrome ) {
+//		browser.webkit = true;
+//	} else if ( browser.webkit ) {
+//		browser.safari = true;
+//	}
+//
+//	jQuery.browser = browser;
+//}
```

Comment:
Is that needed? Could we have `jQuery.browser` instead somehow, which would then disable this piece of code?